app/views.py

```diff
@@ -33,3 +48,11 @@ def page_forbidden(error):
 @url.errorhandler(500)
 def page_server_error(error):
     return render_template('500.html', title='Server Error'), 500
+
+
+def convert_to_readable_time(time):
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
```

Comment:
Don't use month names with slashes. Slashes and dashes are for digits.

If you wish to use the month's name, use spaces.

e.g. `April 8, 2016 12:33 PM`